### DISCUSSION TITLE
Left/Right channel timing alignment issue

### DISCUSSION THREAD

**Discussion by ZeChArtiahSaher** [apohl79/audiogridder#1020] [Help]
I've noticed that with some plugins I get left channel louder than right one. And I did not particularly find a pattern which causes the occurrence specifically. Sometimes the same plugin will load and sound even, and sometimes the channels are desynced. 

Also what's weird is that the issues occur at various instances in various ways when plugins are used. However, if I use element fx (vst host as vst) and load same plugin in there that's supposedly misbehaving there it's all channels are perfectly even

Any ideas what it may be or what should I dump?

**Comment by apohl79** [comment]
Is this something you can reproduce reliably with a particular plugin?

AG is not adjusting the sample values. It passed them into a plugin and back into the DAW. The only exception to this is when your DAW is using 64bit floats for sample values, but you are loading a plugin that is 32bit only (like all AU or VST2 plugins). In this case the sample values will be converted to 32bit, but I can't see how this would affect different channels differently (especially not the volume). Maybe something else is going wrong. If you can reproduce it, can you provide the logs and/or instructions?
